### DISCUSSION TITLE
Will true public teams come back one day?

### DISCUSSION THREAD

**Discussion by typed-sigterm** [community/community#134957] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

> People who are not members of the organization cannot view any teams.

Will the **true** public teams come back one day after [the email storm event](https://github.com/EpicGames/Signup/pull/24)?

For some open-source projects, it is really important to know which teams and who people are in the team. I think providing an option to publicly disclose a team member list to everyone should not lead to such incidents happening again.

(previous discussion thread: #125802)

**Comment by typed-sigterm** [comment]
bump

**Comment by Navyansh999** [comment]
Right now, GitHub does not allow true public teams where non-members can freely see team details inside an organization. After past privacy concerns, team visibility was restricted to protect users from unwanted exposure, so only organization members can view teams and their members. As of now, there’s no official announcement that fully public teams will return. For open-source projects, a practical workaround is to list maintainers and team roles in the repository’s README, contributors page, or project website to maintain transparency without relying on public team visibility.
